dmoj/checkers/floats.py

```diff
@@ -27,7 +28,12 @@ def verify_default(process_float, judge_float, epsilon):
             abs(1.0 - process_float / judge_float) <= epsilon)
 
 
-def check(process_output, judge_output, precision=6, error_mode='default', **kwargs):
+def check(process_output, judge_output, precision=6, high_precision=False, error_mode='default', **kwargs):
```

Comment:
What about precision and other stuff you can control in the [decimal context](https://docs.python.org/3/library/decimal.html#context-objects)?